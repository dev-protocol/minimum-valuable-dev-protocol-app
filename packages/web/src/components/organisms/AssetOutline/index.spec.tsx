import React from 'react'
import { render } from '@testing-library/react'
import { AssetOutline } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')

describe(`${AssetOutline.name}`, () => {
  test('Snapshot', () => {
    const component = render(<AssetOutline metricsAddress="metrics-address" marketAddress="market-address" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
