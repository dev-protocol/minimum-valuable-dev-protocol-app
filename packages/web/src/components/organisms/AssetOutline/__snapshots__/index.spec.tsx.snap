// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetOutline Snapshot 1`] = `
<body>
  <div>
    <div>
      <div
        style="width: 224px; height: 224px; border-radius: 50%; background-color: rgb(47, 128, 237); background-clip: padding-box; border: -2747.89872px solid rgba(47, 128, 237, 0.2);"
      />
    </div>
  </div>
</body>
`;
